contrib/inventory/gce_googleapiclient.py

```diff
@@ -24,51 +25,61 @@
 
 Before using:
 
-- Authentication: this script uses the same authentication as gcloud command line. So, set it up before.
-- Dependencies: it depends mainly on google-api-python-client and docoptcfg. So:
+- Authentication: this script uses the same authentication as
+                  gcloud command line. So, set it up before.
+- Dependencies: it depends mainly on google-api-python-client
+                and docoptcfg. So:
 
 $ pip install google-api-python-client docoptcfg
 
 Usage:
-    {script_name} [options]
+    {script_name} (--billing-account=ACCOUNT_NAME --all-projects | --billing-account=ACCOUNT_NAME | --project=PROJECT...)
+                  [--project=PROJECT...]
```

Comment:
all this complication is just to allow `--all-projects` and `--all-zones` to be ommited?

if so, that is just too complicated (remember that it is the message that the user would see when typing `--help`).

i didn't try, but i think that you can achieve the same thing with:
```
([--all-projects] --billing-account=ACCOUNT_NAME|--project=PROJECT...)
```
and add `[Default: True]` to `--all-projects` on its description on the 'Options:' section.